### PR TITLE
fix: new svg for better performance

### PR DESCRIPTION
**Pull request by PatrickHeneise**
Signed-off-by: Patrick Heneise <patrick@zentered.co>

### CODE REVIEW

File: content/pages/homepage.mdx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 export const meta = {
   title: 'cert-manager',
-  description: 'Cloud native X.509 certificate management for Kubernetes and OpenShift',
+  description:
+    'Cloud native X.509 certificate management for Kubernetes and OpenShift',
   hero: {
     heading: 'Cloud native certificate management',
     description: 'X.509 certificate management for Kubernetes and OpenShift',
-    image: '/images/cert-manager-graphic.png'
+    image: '/images/cert-manager-graphic.svg'
```
[the remark this answers is not in the document]
Comment (SgtCoDFish) [reply content/pages/homepage.mdx:8]: I removed this myself :+1:

**Review by SgtCoDFish — APPROVED** [review]
I'm now a contributor so can't LGTM but I approve now!

**Comment by maelvls** [comment]
Thanks!

/lgtm

File: content/pages/homepage.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (PatrickHeneise) [reply content/pages/homepage.mdx:8]: idk, didn't see the raw images. Sometimes it's good to have the "originals" somewhere, but that's your call.

**Pull request merged**